mods/ts/rules/aircraft.yaml

```diff
@@ -78,12 +78,13 @@ ORCA:
 		RearmBuildings: gahpad, nahpad
 		ROT: 5
 		Speed: 186
+		MoveIntoShroud: false
 	Health:
 		HP: 200
 	Armor:
 		Type: Light
 	RevealsShroud:
-		Range: 10c0
+		Range: 2c0
```

Comment:
see https://www.youtube.com/watch?v=djhoinPs5f0&t=8m35s
I took that sight range from the original rules.ini